src/__tests__/use-app/use-app.test.tsx

```diff
@@ -149,15 +150,16 @@ describe('_app support', () => {
   });
 
   describe('route matching "_app" page', () => {
-    it('throws "page not found" error', async () => {
-      await expect(
-        getPage({
-          nextRoot: __dirname + '/__fixtures__/custom-app-with-gip',
-          route: '/_app',
-        })
-      ).rejects.toThrow(
-        '[next-page-tester] No matching page found for given route'
-      );
+    it('throws "page not found" error page', async () => {
+      const { render } = await getPage({
+        nextRoot: __dirname + '/__fixtures__/custom-app-with-gip',
+        route: '/_app',
+      });
+      render();
+      expect(screen.getByText('404')).toBeInTheDocument();
+      expect(
+        screen.getByText('This page could not be found', { exact: false })
+      ).toBeInTheDocument();
```

Comment:
If we went this direction, we might want to wrap up a test utility function to say "expect this to be error page".